packages/apollo-link-http-common/src/index.ts

```diff
@@ -233,18 +241,18 @@ export const selectHttpOptionsAndBody = (
   };
 };
 
-export const serializeFetchBody = body => {
-  let serializedBody;
+export const serializeFetchParameter = (p, label) => {
```

Comment:
@jaydenseric This is going to be a breaking change for apollo-client-upload

Reply:
Is this OK? I thought this hadn't been released yet so it was an OK interface to break without documenting.

Reply:
It's ok if the change is documented in a changelog entry, and a new version of `apollo-link-http-common` is published with a sember major (breaking change).

Reply:
Ah, I didn't realize Evans had published a non alpha version of the package.

Reply:
Evans, should I just revert the name change or something, or bump to 0.2.0?

Reply:
If you keep the breaking changes, please bump to v1.0.0, breaking changes in semver minor releases are the devil.

Reply:
@glasser We can bump to v1.0.0, since it will make Jayden's life easier. I'm not worried about having the version number bloat and it might force better practice. The plan for apollo-link-common-http was to be a semi private API(without documentation due to overhead) that developers of links can look at if they want to align with apollo-link-http and apollo-link-batch-http. In that goal I think attempting to follow SemVer would be make keeping up with the changes easier

Reply:
Sorry I am wrong ^. Let's do v0.2, since we are pre v1.0 SemVer says that a breaking change should be a [minor version](https://semver.org/#spec-item-4). Also it looks like we should be okay for Jayden's [upload-client](https://github.com/jaydenseric/apollo-upload-client/blob/master/package.json#L38)

Reply:
> [If your software is being used in production, it should probably already be 1.0.0. If you have a stable API on which users have come to depend, you should be 1.0.0. If you’re worrying a lot about backwards compatibility, you should probably already be 1.0.0.](https://semver.org/#how-do-i-know-when-to-release-100)

All 3 things should be assumed to be true for every npm package that can be publically installed. There are only upsides and no downsides to bumping the major version number if there is ever a breaking change.

FB in particular is the absolute worst for this. `react` [jumping from v0.14 to v15](https://reactjs.org/blog/2016/02/19/new-versioning-scheme.html) was an admission of guilt they had it wrong for 14 major versions. [`graphql` being a v0.x](https://github.com/graphql/graphql-js/issues/1005) causes great disruption to the ecosystem every update.